docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -7,11 +7,11 @@ The Instance is the top-level openDAQ(TM) object, serving as a container for the
   * xref:background_info:opendaq_architecture.adoc#instance[Architecture - Instance]
 
 == Creating the default openDAQ(TM) Instance
-Upon creation, it generates a Client device as the default root device. The client device is capable of using the module manager to add function blocks and connect to any device supported by the loaded modules. The client is set as the root device when the instance is created if the developer has not set their own device.
+Upon creation, it generates a Client device as the default root device. The client device is capable of using the module manager to add function blocks and connect to any device supported by the loaded modules. The client is set as the root device when the instance is created if the developer has not set their own device. 
 
 == Creating openDAQ(TM) Instance with Instance Builder
 
-openDAQ(TM) offers an advanced method for configuring an Instance through the Instance Builder. With this, developers can customize the Logger, Scheduler, Module Manager, and Root Device by either defining custom modules or modifying default objects.
+openDAQ(TM) offers an advanced method for configuring an Instance through the Instance Builder. With this, developers can customize the Logger, Scheduler, Module Manager, and Root Device by either defining custom modules or modifying default objects. Default builder values are described in doxygen.
```

Comment:
We should not link to Doxygen within the antora documentation. If the default values are important to this document, there should be a table listing them.